app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules.
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable,
+         :confirmable, :omniauthable
```

Comment:
In this training we will only use:
`database_authenticatable, :registerable, :validatable`
You can add `remenberable` and `trackable` but is not be used